internal/debug/flags.go

```diff
@@ -90,14 +95,15 @@ var (
 
 // Flags holds all command-line flags required for debugging.
 var Flags = []cli.Flag{
-	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag,
+	verbosityFlag, vmoduleFlag, backtraceAtFlag, debugFlag, swaplogpathFlag,
 	pprofFlag, pprofAddrFlag, pprofPortFlag,
 	memprofilerateFlag, blockprofilerateFlag, cpuprofileFlag, traceFlag,
 }
 
 var (
-	ostream log.Handler
-	glogger *log.GlogHandler
+	ostream     log.Handler
+	glogger     *log.GlogHandler
+	swaplogpath string
```

Comment:
This would then be passed via the `swap.New` function

Reply:
fixed and removed from `internal/debug/flags.go`